dash/development/base_component.py

```diff
@@ -265,6 +323,137 @@ def __len__(self):
         return length
 
 
+def schema_is_nullable(type_object):
+    if type_object:
+        if type_object.get('name', None) == 'enum':
+            values = type_object['value']
+            for v in values:
+                value = v['value']
+                if value == 'null':
+                    return True
+        if type_object.get('name', None) == 'union':
+            values = type_object['value']
+            if any([schema_is_nullable(v) for v in values]):
```

Comment:
```suggestion
            if any(schema_is_nullable(v) for v in values):
```

Don't need the duplicate list, generator is better for any since it returns as one is true.